src/mapml-viewer.js

```diff
@@ -621,16 +648,26 @@ export class MapViewer extends HTMLElement {
     }
     if (this._historyIndex === 0) {
       // when at initial state of map, disable back, forward, and reload items
-      this._map.contextMenu._items[0].el.el.disabled = true; // back contextmenu item
-      this._map.contextMenu._items[1].el.el.disabled = true; // forward contextmenu item
-      this._map.contextMenu._items[2].el.el.disabled = true; // reload contextmenu item
+      this._map.contextMenu.toggleContextMenuItems("Back", "disabled"); // back contextmenu item
+      this._map.contextMenu.toggleContextMenuItems("Forward", "disabled");// forward contextmenu item
+      this._map.contextMenu.toggleContextMenuItems("Reload", "disabled"); // reload contextmenu item
     } else {
-      this._map.contextMenu._items[0].el.el.disabled = false; // back contextmenu item
-      this._map.contextMenu._items[1].el.el.disabled = true; // forward contextmenu item
-      this._map.contextMenu._items[2].el.el.disabled = false; // reload contextmenu item
+      this._map.contextMenu.toggleContextMenuItems("Back", "enabled"); // back contextmenu item
+      this._map.contextMenu.toggleContextMenuItems("Forward", "disabled");// forward contextmenu item
+      this._map.contextMenu.toggleContextMenuItems("Reload", "enabled"); // reload contextmenu item
+    }
+  }
+  _reloadButtonState(){
+    if(this.hasAttribute("controls")){
```

Comment:
If the map does not have controls buttons, the _reloadbutton is gone

Reply:
See the code I pushed; if the reload button doesn't exist, it won't enable/disable.  I removed the _reloadButtonState() method and replaced it with this._reloadButton?.enable() / .disable() in the place where the other enabling / disabling is done.   Hopefully because the code is all co-located it will be more "maintainable", at least that's the theory.